README.md

```diff
@@ -6,6 +6,45 @@
 
 ![](https://i.ibb.co/bJCbCf2/202203292020.gif)
 
+## Usage
+
+You can manage tests both from the Testing view and directly within your test files.
+
+### In the Testing View
+
+![Testing view](./img/vitest-extension.png "Testing view")
+
+You can access the extension from the Testing view in the Visual Studio Code sidebar.
+
+The toolbar at the top provides various commands to manage test execution:
+
+- **Refresh Tests**: To reload your test suite, reflecting any new changes.
+- **Run All Tests**: To start testing all cases that are currently visible.
+- **Debug Tests**: To begin a debugging session for the tests.
```

Comment:
The button after debug tests is run tests with coverage

Reply:
Fixed.